rosdep/python.yaml

```diff
@@ -5676,7 +5692,7 @@ python3-flake8:
       packages: [flake8]
   rhel:
     '*': ['python%{python3_pkgversion}-flake8']
-    '7': null
+    '7':
```

Comment:
There are still some unwanted changes here. Even if I were to agree with some of them, they should be done as part of another pull request. Please get the diff down to just your new stanza for the new rosdep rule, thanks :)

Reply:
We specifically use null to indicate that it's unmet. 

From: https://yaml.org/type/null.html
> Note that a null is different from an empty string and that a mapping entry with some key and a null value is valid and different from not having that key in the mapping.

Reply:
Ah ok. Maybe the clean_rosdep_yaml script should be updated then ? Because when I ran it, it removed all the null values.

Reply:
> Ah ok. Maybe the clean_rosdep_yaml script should be updated then ? Because when I ran it, it removed all the null values. Or are you actually not supposed to run this script ? Because an error message in the tests suggested to run it.

Ah, that script may be out-of-date.  I don't know that its been updated to handle the `null` values; those are somewhat new.  If you'd like to contribute a PR to update it, we'd be happy to review!

Reply:
Ok, I'll have a look once I get the chance!